### PR TITLE
Tabs > handling single tab use case

### PR DESCRIPTION
**Pull request by redperk**
This PR adds a check to handle situations where only 1 tab is passed in. If only 1 tab is passed in, it will display that tab at the top and disable the dropdown feature.

### CODE REVIEW

File: src/components/Tabs.js [hunk 1]
```diff
@@ -26,9 +26,18 @@ class Tabs extends React.Component {
 
   state = {
     selectedTab: this.props.selectedTab || 0,
-    showMenu: false
+    showMenu: false,
+    singleTab: false
   };
 
+  componentDidMount () {
+    if (this.props.tabs.length === 1) {
```
Comment (paniclater) [inline src/components/Tabs.js]: What made you want to use state here instead of relying on the prop.tabs.length check on its own?

Comment (redperk) [reply src/components/Tabs.js]: That's a good point. I thought I was going to need it in more places, but I'm only using it once. I'll make the change.

**Comment by paniclater** [comment]
This looks good, just has the conflicts to be taken care of. I do want to note that it looks like to me if the state had been used as is there would have been bugs because the state wouldn't have been updated on componentDidUpdate, only on componentDidMount. Issues like this lead me to recommend that wherever possible we avoid assigning props to state and managing that reconciliation process. 

Thanks for the updates

**Comment by redperk** [comment]
Closing this since Joe already had a fix for this that he had forgotten to merge in.

**Pull request closed without merging**